sw/ground_segment/tmtc/server.ml

```diff
@@ -438,7 +435,7 @@ let send_aircraft_msg = fun ac ->
                   "temp", f a.temp;
                   "bat", f a.bat;
                   "amp", f a.amp;
-                  "energy", PprzLink.Int a.energy] in
+                  "charge", f a.charge] in
```

Comment:
why charge here instead of keeping energy ? also unit in message definition is not matching

Reply:
Energy is power x time in Wh and charge is current x time in Ah. In the code, energy was sometimes wrongly used as charge. I corrected this, renaming everything that was measuring or displaying charge to be called charge. I also added a true measure of energy.

Here, charge was being sent so I updated it. So either both the name and the unit were wrong or the server was sending the wrong value. Do you know if this should have been energy or charge? Doesn't look like this value was being used anywhere.

This also reminds me I forgot to make a pull request for the pprzlink side of this.